src/script/lua_api/l_client.h

```diff
@@ -105,6 +108,9 @@ class ModApiClient : public ModApiBase
 	// get_csm_restrictions()
 	static int l_get_csm_restrictions(lua_State *L);
 
+	// store_texture(texture, path)
+	static int l_store_texture(lua_State *L);
```

Comment:
This may be in a 2nd PR, about storing textures.

Reply:
Will do.